config/initializers/backport_pg_10_support_to_rails_4.rb

```diff
@@ -0,0 +1,45 @@
+require 'active_record/connection_adapters/postgresql/schema_statements'
+
+#
+# Monkey-patch the refused Rails 4.2 patch at https://github.com/rails/rails/pull/31330
```

Comment:
Did this go to 5.0? if yes I will remove it there when this got merged. Anyway 👍